### ISSUE TITLE
Cannot install plone.app.caching when it is not available on site creation

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/plone.app.caching#139]
1. Create a Plone 6 site with only Products.CMFPlone.
2. Add plone.app.caching to the packages and restart.
3. You cannot install it from the Add-ons control panel.

Workaround: go the portal_setup in the ZMI and install it there.

So:

- [x] Remove `plone.app.caching` from the hidden products in `Products/CMFPlone/factory.py`.
- [x] Create an uninstall profile.
- [x] For good measure change the custom import step into a `post_handler` for the default profile.  That is a separate point, but can easily be handled at the same time.

Best do this only for Plone 6.1, so this would need a branch 3.x to use on Plone 6.0.

**Comment by yurj** [comment]
Can we close this issue?

**Comment by mauritsvanrees** [comment]
Yes, I just verified it in 6.2 and this can be closed.  See the commits and PRs linked around 12 September 2024.
I have ticked the checkboxes for good measure.
